pyscreeze/__init__.py

```diff
@@ -102,6 +102,28 @@
     else:
         raise
 
+isWaylandSession = False
+if sys.platform == 'linux':
+    try:
+        # ref: https://unix.stackexchange.com/questions/202891/how-to-know-whether-wayland-or-x11-is-being-used#comment1133584_371164
+        session_string = subprocess.check_output("loginctl show-session $(loginctl show-user $(whoami) -p Display --value) -p Type --value", shell=True, stderr=subprocess.STDOUT).decode().strip()
```

Comment:
Might want to leave a comment mentioning that this relies on the host being systemd based.

Reply:
Fair point; that probably belongs in the commit message. How would you revise the commit message to reflect that nuance?

Reply:
Something along the lines of "This currently relies on processing information from loginctl provided by systemd in order to validate whether we are inside of a wayland session, which should work in most setups today."